packages/addon/src/background.ts

```diff
@@ -261,6 +263,13 @@ browser.runtime.onMessage.addListener(async (message) => {
       });
       break;
 
+    case SIGN_IN_COMPLETE:
+      console.log(
+        `[onMessage] background.ts received SIGN_IN_COMPLETE. Telling menu.ts to close tab.`
+      );
+      await closeLoginTab();
+      break;
+
```

Comment:
Closing the tab after authentication, part 3/5

When `background.ts` receives the "ok to close tab" message from the bridge, it calls a function in `menu.ts`.

(`menu.ts` has to be the one to close the tab, since it is the one that created it)